tests/test_lightningd.py

```diff
@@ -19,6 +21,8 @@
     logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
 logging.info("Tests running in '%s'", TEST_DIR)
 
+def to_json(arg):
```

Comment:
I'm pretty sure this is only a noop, the RPC interface takes care of serializing the arguments.